platform/platform.go

```diff
@@ -24,17 +24,32 @@ type Platform interface {
 	Regrade([]RegradeSpec) error
 }
 
+type RemotePlatform interface {
+	Platform
+	Ping() error
```

Comment:
Platform could have a `Ping() error` too, and have standalone platform return nil (or contact k8s). Then we could maintain a single interface.